scripts/SANS/sans/algorithm_detail/batch_execution.py

```diff
@@ -1129,47 +1154,33 @@ def get_all_names_to_save(reduction_packages, save_can):
         reduced_lab_sample = reduction_package.reduced_lab_sample
         reduced_hab_sample = reduction_package.reduced_hab_sample
 
-        transmission = reduction_package.unfitted_transmission
-        transmission_can = reduction_package.unfitted_transmission_can
-        try:
-            # Does not always work for event slice data as transmission
-            # workspaces can get deleted.
-            # This is a temporary fix to allow file saving with
-            # event sliced data.
-            trans_name = '' if not transmission else transmission.name()
-        except RuntimeError:
-            trans_name = ''
-            Logger("SANS").notice("Transmission run not found. Not saving to file.")
-        try:
-            transCan_name = '' if not transmission_can else transmission_can.name()
-        except RuntimeError:
-            transCan_name = ''
-            Logger("SANS").notice("Can transmission run not found. Not saving to file.")
+        trans_name = get_transmission_names_to_save(reduction_package, False)
+        transcan_name = get_transmission_names_to_save(reduction_package, True)
 
         if save_can:
             if reduced_merged:
-                names_to_save.append((reduced_merged.name(), trans_name, transCan_name))
+                names_to_save.append((reduced_merged.name(), trans_name, transcan_name))
```

Comment:
might be better as `trans_can_name` to follow snake case

Reply:
Fixed now